src/Mod/Path/PathScripts/PathMillFace.py

```diff
@@ -106,6 +106,7 @@ def areaOpShapes(self, obj):
 
         if obj.Base:
             PathLog.debug("obj.Base: {}".format(obj.Base))
+            self.removalshapes = list()
```

Comment:
sticks out like a .... wanna use `[]` like all the other initializers?

Reply:
Thanks for the review.

Not sure how to fully interpret the question and all that might go with it, so my simple answer is, no, sir.

Reply:
lol - what I meant was could you please use
`self.removalshapes = []`
to initialise